Detectors/TPC/reconstruction/test/testTPCCATracking.cxx

```diff
@@ -0,0 +1,61 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file testTPCCATracking.cxx
+/// \brief This task tests the TPC CA Tracking library
+/// \author David Rohr
+
+#define BOOST_TEST_MODULE Test TPC CATracking
+#define BOOST_TEST_MAIN
+#define BOOST_TEST_DYN_LINK
+#include <boost/test/unit_test.hpp>
+#include "DataFormatsTPC/Constants.h"
+#include "DataFormatsTPC/Helpers.h"
+#include "DataFormatsTPC/TrackTPC.h"
+#include "DataFormatsTPC/ClusterNative.h"
+#include "TPCReconstruction/TPCCATracking.h"
+
+#include <vector>
+#include <iostream>
+#include <iomanip>
+
+using namespace o2::dataformats;
+
+namespace o2 {
+namespace TPC {
+
+  /// @brief Test 1 basic class IO tests
+  BOOST_AUTO_TEST_CASE(CATracking_test1)
```

Comment:
formatting issues, please apply the clang format directly on the new file rather than in a later commit